source/agora/node/Ledger.d

```diff
@@ -928,7 +921,7 @@ public class Ledger
     public Hash getValidatorRandomSeed (in Height height) nothrow
```

Comment:
How about removing this function in this PR. This function is not used anymore anyway.

Reply:
I added a commit to remove this.